examples/tutorials/01_basics/01_hello_world.py

```diff
@@ -16,22 +16,15 @@
 
 '''Hello World
 
-Make Cozmo say 'Hello World' in this simple Cozmo SDK example program.
+Make Cozmo say 'Hello World' and then wait for him to finish speaking
```

Comment:
I like the previous text more without the "and then wait for him to finish speaking".